modules/lang-painless/src/main/java/org/opensearch/painless/antlr/PainlessParser.java

```diff
@@ -30,6 +30,7 @@
  * Modifications Copyright OpenSearch Contributors. See
  * GitHub history for details.
  */
+
```

Comment:
@camerondurham could you please revert this change please? (it is not harmful but adds a noise to history), thank you

Reply:
For sure, sorry, let this slip in. Removed from files changed.